Gateway/Request/PaymentAPMBuilder.php

```diff
@@ -4,11 +4,11 @@
 use Magento\Framework\UrlInterface;
 use Magento\Payment\Gateway\Helper\SubjectReader;
 use Magento\Payment\Gateway\Request\BuilderInterface;
-use Omise\Payment\Model\Config\Offsite\Alipay;
-use Omise\Payment\Model\Config\Offsite\Internetbanking;
-use Omise\Payment\Observer\OffsiteInternetbankingDataAssignObserver;
+use Omise\Payment\Model\Config\Alipay;
+use Omise\Payment\Model\Config\Internetbanking;
+use Omise\Payment\Observer\InternetbankingDataAssignObserver;
 
-class PaymentOffsiteBuilder implements BuilderInterface
+class PaymentAPMBuilder implements BuilderInterface
```

Comment:
@jacekstan (optional) just quick look, APM already stands for Alternative Payment Method (maybe, not 100% sure lol)
PaymentAPM is a bit redundant. How about just `APMBuilder`?

Reply:
other builders have Payment word in the same folder..
almost all of them...
Actually now thinking.. should we add Payment to one missing name, or remove from all of them...
and that not an error.. you have a lot in different sectors of life.. that you repeat element from abbrev. just to show what that entire name construction is about. Even if any letter of that abbreviation stands for already used word in that name (not sure if you know what I mean.)

Reply:
Shall we remove all then?
I was prefixing it as `Payment` with no strong reason, just to clarify that the class is for payment.
But after more methods coming, I kinda have a second thought that probably we can just remove it all, since everything in `Gateway` folder will always relate to the Payment function.